src/effects/backends/builtin/beatgrindeffect.h

```diff
@@ -38,6 +39,7 @@ class BeatGrindState : public EffectState {
     int readSamplePos;
     int writeSamplePos;
     bool isRecording;
+    double mixPrev;
```

Comment:
Please spell out words if they aren't too long.
```suggestion
    double mixPrevious;
```